molecule/commands.py

```diff
@@ -32,7 +32,7 @@
 
 import molecule.utilities as utilities
 import molecule.validators as validators
-from provisioners import Ansible
+from molecule.core import Molecule
```

Comment:
Haha, not so much holly war though: http://docs.openstack.org/developer/hacking/#imports